config/jobs/image-pushing/k8s-staging-e2e-test-images.yaml

```diff
@@ -198,6 +198,16 @@ postsubmits:
             - name: WHAT
               value: "echoserver"
     - name: post-kubernetes-push-e2e-glusterdynamic-provisioner-test-images
+      rerun_auth_config:
+        github_team_slugs:
+          - org: kubernetes
+            slug: release-managers
+          - org: kubernetes
+            slug: test-infra-admins
+        github_users:
+          - aojea
+          - chewong
+          - claudiubelu
```

Comment:
Thanks for doing this.  More of a note-to-self that I'd like to turn this into a GitHub team